site/pages/smart-contracts/modular-account-v2/managing-ownership.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Managing ownership
+description: Managing ownership on your Modular Account V2
+---
+
+# Managing Ownership
+
+You can add an owner to your account, or transfer ownership of your account with Modular Account V2. Adding an owner has the same flow as adding a session key with no root permissions.
```

Comment:
If the below example transfers ownership (i.e. replaces the original owner), relating to my previous comment, I think it might make sense to not talk about being able to add an owner here above the example. Maybe it should go below the example like "If you want to add an additional owner without removing the original owner, you can do so by using session keys. (link to example)"